### PR TITLE
TM-3906 AIM box movement fix

### PR DESCRIPTION
**Pull request by andrews-john**
First pass at a fix

### CODE REVIEW

File: src/sass/_agendaItemLegsForm.scss [hunk 1]
```diff
@@ -6,6 +6,11 @@
   display: grid;
   grid-template-columns: 90px repeat(10, auto);
   align-items: center;
+  max-width: 720px;
```
Comment (SidelineCory24) [inline src/sass/_agendaItemLegsForm.scss:9]: We are not looking to change the size of the entire timeline box. Instead we want to keep it the same size and prevent it from moving once 8+ legs are added.

**current size of aitl box on dev branch**
image

**what your pr is changing the size to** 
image

Comment (SidelineCory24) [reply src/sass/_agendaItemLegsForm.scss:9]: Issue/edge to be addressed: below you'll see that there is not screen movement when the leg is added. However when the second and third leg in the video are added the screen begins to move.

https://user-images.githubusercontent.com/55964163/215473322-e7102755-b0d0-4ddd-b082-107c194d4df9.mp4

**Comment by elizabeth-jimenez** [comment]
@andrews-john updated with dev branch, so make sure to pull - a recent api merge would break EAS unless dev pulled

File: src/sass/_agendaItemMaintenance.scss [hunk 2]
```diff
@@ -229,6 +229,37 @@
...
+        max-width: 1025px;
+      }
+      @media screen and (max-width: 2250px) {
+        color: pink;
+        max-width: 975px;
+      }
+      @media screen and (max-width: 2150px) {
+        color: brown;
+        max-width: 925px;
+      }
```
Comment (elizabeth-jimenez) [inline src/sass/_agendaItemMaintenance.scss:245]: Nice tactic to see the changes 🙌

**Comment by andrews-john** [comment]
Branch has been replaced by https://github.com/MetaPhase-Consulting/State-TalentMAP/pull/2455

**Pull request closed without merging**